dist/dev/css/main-index.css

```diff
@@ -5,4 +5,87 @@
 
 body {
     margin: 0;
+}
+
+.col-lg-1, .col-lg-2, .col-lg-3, .col-lg-4, .col-lg-5, .col-lg-6, .col-lg-7, .col-lg-8, .col-lg-9, .col-lg-10, .col-lg-11, .col-lg-12,
```

Comment:
Also, why are you hard coding your cols? I'm not a CSS expert here, but why does this need to be done in CSS and not JSS?